src/components/pages/group/edit/group-rules/group-rules-table/group-rules-row.tsx

```diff
@@ -2,14 +2,16 @@ import { Button } from "@chakra-ui/core";
 import useTranslation from "@configs/i18n/useTranslation";
 import React from "react";
 
+import ReadMore from "./read-more";
+
 const GroupRulesRow = ({ itemDetails, onDelete }) => {
   const { t } = useTranslation();
   const { name, value } = itemDetails;
 
   return (
     <tr>
       <td>{name}</td>
-      <td>{value}</td>
+      <td>{name === "spatialRule" ? <ReadMore value={value} /> : value}</td>
```

Comment:
In table <ReadMore/> component has Cumulative Layout Shift (CLS) also there;s possiblity this might create problem for screen readers as well this can be dealt very simply like below

```jsx
<td>
  <div className="elipsis">{value}</div>
</td>
```